examples/models/core/deepseek_v3/deployment-guide-for-trt-llm-deepseek-r1.md

```
@@ -0,0 +1,377 @@
+    --tp_size 8 \
+    --ep_size 8 \
+    --trust_remote_code \
+    --extra_llm_api_options ${EXTRA_LLM_API_FILE}
+```
+
+After the server is set up, the client can now send prompt requests to the server and receive results.
+
+## Configs and Parameters
+
+These options are used directly on the command line when you start the `trtllm-serve` process.
+#### `--tp_size`
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)